python/test/unit/language/test_conversions.py

```diff
@@ -76,6 +76,10 @@ def launch_exhaustive_populate(dst_dtype, offset, numel, force_odd, output_bits,
     assert(numel % BLOCK_SIZE == 0)
     dst = torch.empty((numel,), dtype=matching_int(dst_dtype), device='cuda')
     exhaustive_populate[(numel // BLOCK_SIZE,)](triton.reinterpret(dst, dst_dtype), offset, BLOCK_SIZE, force_odd, output_bits, max_repr)
+    # 0x80 in float8e4b8 or float8e5b16 represents inf/nan. We don't to have that
+    # as input to the conversion kernels.
```

Comment:
looks like there is word missing in the second sentence. Could you explain why we don't want to test this input value?

Reply:
Sorry. I meant "We don't need to have that as input to the conversion kernels.". This just follows the existing scheme of generating the input data. In kernel `exhaustive_populate` which generates the input data, it was done in a way that all inf/nan were removed with the following line:
```
vals = tl.where(avals <= max_repr, vals, 0)
```